BuildingBlocks/src/BuildingBlocks.API/Serilog/SensitiveDataEnricherExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Serilog.Enrichers.Sensitive;
+
+namespace Backbone.BuildingBlocks.API.Serilog;
+public static class SensitiveDataEnricherExtensions
+{
+    private const string MASKED_DATA_PLACEHOLDER = "{maskedData}";
+
+    public static void AddSensitiveDataMasks(this SensitiveDataEnricherOptions options)
+    {
+        options.MaskValue = MASKED_DATA_PLACEHOLDER;
+
+        options.MaskingOperators.Add(new IdentityAddressMaskingOperator());
+        options.MaskingOperators.Add(new BackboneIdMaskingOperator("DVC", 17));
```

Comment:
```suggestion
        options.MaskingOperators.Add(new BackboneIdMaskingOperator(DeviceId.PREFIX, DeviceId.MAX_LENGTH));
```
(You have to make the `PREFIX` constant public in order to do that. But I think that's okay)

Reply:
Changes made.